csrc/multidevice/lower_communication.cpp

```diff
@@ -36,6 +83,12 @@ inline at::Tensor createDummyTensor(at::Tensor reference) {
   return at::empty_like(reference, reference.options());
 }
 
+inline at::Tensor createDummyTensor(
+    at::Tensor reference,
+    BinaryOpType op_type) {
+  return createDummyTensor(reference).fill_(getInitialValue<float>(op_type));
```

Comment:
out of curiosity, why is `getInitialValue` called with `float`?

Reply:
There is no particular reason, if I'm correct the type doesn't matter here since the value inside `fill_` will be casted to the tensor type

Do you have a better suggestion ?

Reply:
no, I was just curious